vllm/spec_decode/spec_decode_worker.py

```diff
@@ -408,7 +408,18 @@ def execute_model(
         disable_all_speculation = self._should_disable_all_speculation(
             execute_model_req)
         num_lookahead_slots = execute_model_req.num_lookahead_slots
-
+        all_prompt = None
+        atleast_one_prompt = False
+        for sgm in execute_model_req.seq_group_metadata_list:
+            all_prompt = (sgm.is_prompt if all_prompt is None else all_prompt
```

Comment:
@sroy745 Can we set all_prompt to True initially and just simplify this condition? I don't think we will have an empty list here